packages/test-utils/src/mocks/repair/v1/data.ts

```diff
@@ -0,0 +1,38 @@
+import faker from "faker/locale/en";
+
+export interface Repair {
+  id: string;
+  dateRaised: string;
+  description: string;
+  lastUpdated: string | null;
+  owner: string;
+  priority: string;
+  property: string;
+  propertyPostCode: string | null;
+  propertyReference: string;
+  reference: number;
+  status: string;
+  tradeCode: string;
+  tradeDescription: string;
+}
+
+export const generateMockRepairWorkOrdersV1 = (data: Partial<Repair> = {}): Repair => ({
+  id: faker.datatype.uuid(),
+  reference: faker.datatype.number(123456789),
+  dateRaised: faker.date.past().toISOString(),
```

Comment:
Try avoid using `past`, we've had inconsistent results with it. especially when using snapshots.
It might be better to do `faker.date.between("2000-01-01", "2020-01-01").toISOString()`